admin-dev/themes/new-theme/js/pages/product/mixins/is-attribute-selected.ts

```diff
@@ -35,12 +36,12 @@ export default {
      *
      * @returns {boolean}
      */
-    isSelected(attribute, attributeGroup, attributeGroups) {
+    isSelected(attribute: Record<string, any>, attributeGroup: Record<string, any>, attributeGroups: Record<string, any>) {
```

Comment:
Maybe it would be more readabale if we had dedicated types for Attribute and AttributeGroup?
```suggestion
    isSelected(attribute: Attribute, attributeGroup: AttrbuteGroup, attributeGroups: Record<number, AttributeGroups>) {
```

It would be more explicit that the array must be associated with `AttributeGroup.id`